packages/server/email/components/SummaryEmail/ExportToCSV.tsx

```diff
@@ -116,53 +169,119 @@ class ExportToCSV extends Component<Props> {
 
     const rows = [] as CSVRetroRow[]
     reflectionGroups!.forEach((group) => {
-      const {reflections, tasks, title, voteCount: votes} = group
+      const {reflections, tasks, title, voteCount: votes, thread} = group
       tasks.forEach((task) => {
         rows.push({
-          title: title!,
+          reflectionTitle: title!,
+          author: task!.createdByUser!.preferredName,
```

Comment:
let's use optional chaining here instead `task?.createdByUser?.preferredName ?? 'Anonymous'`. In the future, to be GDPR compliant, we may have to delete everything a user ever did, which means the `createdByUser` object could be blank & with the `!` typescript won't be able to warn us.